src/router.ts

```diff
@@ -28,7 +29,13 @@ const routes: any[] = [
   {
     path: '/',
     name: 'home',
-    component: domains[domainName] ? Space : Home
+    component: Home,
+    beforeEnter(to, from, next) {
+      if (domains[domainName]) {
```

Comment:
Would be great if there is a way so we don't redirect user from custom domain like vote.balancer.fi to vote.balancer.fi/#/balancer